apps/evalite-docs/src/content/docs/guides/traces.mdx

```diff
@@ -37,8 +37,9 @@ evalite("My Eval", {
         },
       ],
       usage: {
-        completionTokens: result.completionTokens,
-        promptTokens: result.promptTokens,
+        inputTokens: result.inputTokens,
+        outputTokens: result.outputTokens,
+        totalTokens: result.totalTokens,
```

Comment:
Thought I'd include this one since it's not necessarily the sum of input and output tokens.